src/test/codegen/consts.rs

```diff
@@ -10,11 +10,11 @@
 // CHECK: @STATIC = {{.*}}, align 4
 
 // This checks the constants from inline_enum_const
-// CHECK: @{{[0-9]+}} = {{.*}}, align 2
+// CHECK: @alloc{{[0-9]+\.[0-9]+}} = {{.*}} zeroinitializer, align 2
```

Comment:
I think we should hardcode the `@alloc` number if it's deterministic (usually the case if it's small like `1` or `2`.

Reply:
added in 9e9837627f712570053cf7042be66d6280a1c6b3

Reply:
the number was 5.0 here

Reply:
with the changes in https://github.com/rust-lang/rust/pull/69155/commits/8fbc283016b7c5123d312e4aa7cf534b6ec9638b there were no more decimals involved.  These transitioned to integer values in the strings.